.openpublishing.redirection.json

```diff
@@ -929,6 +929,86 @@
         "source_path": "docset/virtual-directory-module/medv/new-medvworkspace.md",
         "redirect_url": "/powershell/module/Microsoft.Medv.Administration.Commands.WorkspacePackager/new-medvworkspace",
         "redirect_document_id": false
+        },
+        {
+        "source_path": "docset/windows/wsus/Add-WsusComputer.md",
+        "redirect_url": "/powershell/module/updateservices/Add-WsusComputer",
```

Comment:
Need to include the version moniker at the end of the redirect_url.

"redirect_url": "/powershell/module/updateservices/Add-WsusComputer?view=win10-ps",